### ISSUE TITLE
fresh install redis-server won't start

### ISSUE THREAD

**Issue by pjv** [WordOps/WordOps#111]
If you guys need me to provide all the info in the issue template, tell me and i will.

Fresh Ubuntu 18.04.3 LTS VPS
Fresh WordOps v3.9.7
sudo wo stack install
sudo wo site create example.com --wpredis --php73

...after which:

```
Adding repository for PHP, please wait...
Adding repository for Redis, please wait...
Updating apt-cache, please wait...
Installing packages, please wait...
Restart : php7.3-fpm[OK]
Restart : redis-server[Failed]
```
It took me a bit of noodling around to figure out that the reason the redis server wouldn't start is because the `redis` user could not read the config file at `/etc/redis/redis.conf` whose permissions looked like:

`-rw-r----- 1 root root 61K Aug  8 11:55 redis.conf`

changing the group to `redis` fixed the problem

`sudo chgrp redis /etc/redis/redis.conf`

**Comment by VirtuBox** [comment]
Thanks for you report, I will fix this issue asap.

**Comment by VirtuBox** [comment]
Issue has been fixed with release [v3.9.7.1](https://github.com/WordOps/WordOps/releases/tag/v3.9.7.1)

**Comment by nsgoyat** [comment]
@VirtuBox Are you sure it has been fixed? I am still facing this issue on my recently configured servers.

**Comment by VirtuBox** [comment]
Hello @nsgoyat,
thanks for your report. 
After checking again the section related to redis configuration, I confirm there was an error in the previous fix. I'm working on it

**Comment by VirtuBox** [comment]
Issue has been fixed with release [v3.9.7.2](https://github.com/WordOps/WordOps/releases/tag/v3.9.7.2)
